.eslintignore

```diff
@@ -11,3 +11,6 @@
 integration/helpers/deno-template
 packages/remix-deno
 templates/deno
+
+# IDK What's up with this file...
+decisions/0007-remix-on-react-router-6-4-0.md
```

Comment:
This should be able to go away now if you merge latest `dev` as the issues are fixed